experimental-design/ENZYMES.tsv

```diff
@@ -0,0 +1,21 @@
+NE=2-iodobenzoate; CS='(?<=W)'
+NE=Arg-C; CS='(?<=R)(?\!P)'
+NE=Asp-N; CS='(?=[BD])'
+NE=Asp-N ambic;  CS='(?=[DE])'
+NE=CNBr; CS='(?<=M)'
+NE=Chymotrypsin;  CS='(?<=[FYWL])(?\!P)'
+NE=Formic acid; CS='((?<=D))|((?=D))'
+NE=Lys-C;  CS='(?<=K)(?\!P)'
+NE=Lys-C/P; CS='(?<=K)'
+NE=PepsinA; CS='(?<=[FL])'
+NE=TrypChymo; CS='(?<=[FYWLKR])(?\!P)'
+NE=Trypsin; CS='(?<=[KR])(?\!P)'
+NE=Trypsin/P; CS='(?<=[KR])'
+NE=V8-DE; CS='(?<=[BDEZ])(?\!P)'
+NE=V8-E; CS='(?<=[EZ])(?\!P)'
+NE=glutamyl endopeptidase; CS='(?<=[^E]E)'
+NE=leukocyte elastase; CS='(?<=[ALIV])(?!P)'
+NE=proline endopeptidase; CS='(?<=[HKR]P)(?!P)'
+
```

Comment:
Glu-C
Lys-N
Elatastase (Cleaves at C-Terminus of Ala, Val, Ser, Gly, Leu and Ile)

Reply:
@trishorts after first PR, We can increase the coverage.